mobly/controllers/android_device.py

```diff
@@ -917,6 +917,12 @@ def load_snippet(self, name, package):
         client. E.g. `name='maps'` attaches the snippet client to
         `ad.maps`.
       package: string, the package name of the snippet apk to connect to.
+      instrument_options: dict[str, str], the Android am instrument options used
+        for controlling the `onCreate` process of the app under test. Note that
```

Comment:
is there a specific set of options that's valid here?

Reply:
I didn't find it and I think it is not documented in official doc.